docs/content/commands/npm-access.md

```diff
@@ -91,12 +89,17 @@ Management of teams and team memberships is done with the `npm team` command.
 <!-- AUTOGENERATED CONFIG DESCRIPTIONS START -->
 <!-- automatically generated, do not edit manually -->
 <!-- see lib/utils/config/definitions.js -->
-#### `registry`
+#### `json`
 
-* Default: "https://registry.npmjs.org/"
-* Type: URL
+* Default: false
+* Type: Boolean
 
-The base URL of the npm registry.
+Whether or not to output JSON data, rather than the normal output.
+
+* In `npm pkg set` it enables parsing set values with JSON.parse() before
```

Comment:
Should this be a description of `npm access set` vs. `npm pkg set`?

Reply:
It will be nice when we have command specific config items.